<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>369</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user_widget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>6</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addsemok_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>109</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>신규등록 추가</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="hun_semokcancel_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hun_semoksave_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>57</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>헌금 항</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_semok_reg_widget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="registed_semok_tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string>기존 등록된 세목</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="new_semok_tableWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>신규등록 목</string>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>신규 헌금 세목  등록</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hang_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>53</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>83</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>신규등록 세목</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>54</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>57</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>회계 구분</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>84</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>헌금 목</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mok_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="semok_view_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택행 수정</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="semok_delete_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>204</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택행 삭제</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>addsemok_button</tabstop>
  <tabstop>hun_semoksave_button</tabstop>
  <tabstop>hun_semokcancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>addsemok_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>hunsemok_reg_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hun_semoksave_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>semokfile_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hun_semokcancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>semokfile_save_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>semok_delete_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>semok_view_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>hunsemok_modify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>hunsemok_reg_input()</slot>
  <slot>semokfile_save()</slot>
  <slot>semokfile_save_cancel()</slot>
  <slot>hunsemok_modify()</slot>
  <slot>delete_selected_row()</slot>
 </slots>
</ui>
